internal/state/api/channels.go

```diff
@@ -59,6 +60,88 @@ func (s *Server) getGuildChannels(w http.ResponseWriter, r *http.Request, p http
 	return s.writeTerms(w, cs)
 }
 
+func (s *Server) setGuildChannels(w http.ResponseWriter, r *http.Request, p httprouter.Params) error {
+	guild, err := guildParam(p)
+	if err != nil {
+		return xerrors.Errorf("read guild param: %w", err)
+	}
+
+	var channelsData = make(map[string]interface{})
```

Comment:
same below. try `map[string]byte[]` if works

Reply:
does not work for this, left as it is